### DISCUSSION TITLE
egzumer / uv-k5-firmware-custom / French manual available with google translate

### DISCUSSION THREAD

**Discussion by reminoo** [egzumer/uvk5-chirp-driver#37] [Show and tell]
Good morning,
I have just finished translating and formatting the manual in .odt. I would like to give it to the community.
I made it for me. so it's possible that you won't like it. I would like to thank you for your software. 73
[Welcome to the uv-k5-firmware-custom wiki! FR.odt](https://github.com/egzumer/uvk5-chirp-driver/files/14444151/Welcome.to.the.uv-k5-firmware-custom.wiki.FR.odt)

**Comment by joc2** [comment]
nice idea... i will take a look.
